requirements.txt

```diff
@@ -3,10 +3,10 @@ aiobotocore[awscli]==1.1.2
 boto3==1.14.44
 botocore==1.17.44
 # Pinned for production DEA run
-fc==1.3.2
-wofs==1.6.1
-odc-aws
-odc-index
+fc==1.3.3
```

Comment:
@alexgleith Please note this package version change. I'm not sure if it *should* be changed, but noticed that 1.3.3 is now released here: https://packages.dea.ga.gov.au/fc/

Reply:
All that's changed in `fc` at the moment is the CI setup. It would be better to leave it at 1.3.2 until we have a chance to test.

https://github.com/GeoscienceAustralia/fc/releases

Reply:
Yes, please don't change.

Changes need a change control board request.

Reply:
Package version has been reverted in cdddcff1abc29cad94aea01879334b2e3fac4eb7